install-script.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+PWD=$(pwd)
+
+# first, backup any existing scripts if they are not symlinks
```

Comment:
Why skip the backup if they're symlinks? What if the user had carefully chosen an interesting place to symlink their `.vim` to?